lib/tasks/manageiq/ui_tasks.rake

```diff
@@ -92,6 +90,8 @@ namespace :webpack do
   end
 
   task :paths do
```

Comment:
I think we should just make this a prerequisite of the task using the normal mechanisms:

```ruby
task :paths => ["ui:load_app_inflectors", "ui:load_asset_engine_inflectors"] do
```

* * *

Alternatively, we could also make a `task` under the `:ui` namespace that does this:

```ruby
task :load_inflectors => ["ui:load_app_inflectors", "ui:load_asset_engine_inflectors"]
```

And then the above turns into:

```ruby
task :paths => "ui:load_inflectors" do
```